index.js

```diff
@@ -1,4 +1,7 @@
-var fs = require('fs');
-var sysPath = require('path');
+const fs = require('fs');
+const sysPath = require('path');
 
-module.exports = fs.readFileSync(sysPath.join(__dirname, 'require.js'), 'utf8');
+const readFile = name => fs.readFileSync(sysPath.join(__dirname, name), 'utf8');
+const requireJs = readFile('require.js');
```

Comment:
Which is highly unlikely given that brunch 1.x.x which does support node 0.12 depends on cjs `~0.2.0` — https://github.com/brunch/brunch/blob/b8b25bc2c0e6dd21a8ad3d2a8e0157e770f3d3d2/package.json#L43

Brunch 2 only does node 4 & 5, so should be good